src/auth/src/credentials/idtoken.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2024 Google LLC
```

Comment:
nit: copyright year is wrong, probably cut&paste error?

Reply:
solved